html-aam/index.html

```diff
@@ -9270,6 +9270,201 @@ <h4 id="att-colspan">`colspan`</h4>
+                  <li>The element is a descendant of the `popover` it is associated with.</li>
+                </ul>
+                <p class="note">
+                  A button that represents a <a data-cite="html/forms.html#concept-submit-button">submit button</a> or is in the
+                  <a data-cite="html/input.html#reset-button-state-(type=reset)">reset state</a> with a [=button/form owner=] cannot invoke a command.
+                </p>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+        <h4 id="att-command-dialogs">`command` <span class="att-context">(in the Close and Show Modal states)</span></h4>
+        <table class="data" aria-labelledby="att-command-dialogs">
+          <tbody>
+            <tr>
+              <th>HTML Specification</th>
+              <td>`command`</td>
+            </tr>
+            <tr>
+              <th>Element(s)</th>
+              <td>
+                <a data-cite="html/button.html#attr-input-command">`button`</a>
+                <span class="el-context"
+                  >(`command` in the <a data-cite="html/input.html#attr-button-command-close-state">Close state</a>) and
+                  <a data-cite="html/input.html#attr-button-command-show-modal-state">Show Modal state</a>)</span
+                >
+              </td>
+            </tr>
+            <tr>
+              <th>[[WAI-ARIA-1.2]]</th>
+              <td>
+                <div class="general">A `command` attribute in the `close` and `show-modal` states provide no additional accessibility mappings to the `button` element.</div>
+              </td>
+            </tr>
+            <tr>
+              <th>
+                <a href="https://msdn.microsoft.com/en-us/library/dd373608%28v=VS.85%29.aspx">MSAA</a> + <a href="http://accessibility.linuxfoundation.org/a11yspecs/ia2/docs/html/">IAccessible2</a>
+              </th>
+              <td>
+                <div class="general">Not mapped</div>
+              </td>
+            </tr>
+            <tr>
+              <th><a href="https://msdn.microsoft.com/en-us/library/ms726297%28v=VS.85%29.aspx">UIA</a></th>
+              <td>
+                <div class="general">Not mapped</div>
+              </td>
+            </tr>
+            <tr>
+              <th><a href="https://gnome.pages.gitlab.gnome.org/atk/">ATK</a></th>
+              <td>
+                <div class="general">Not mapped</div>
+              </td>
+            </tr>
+            <tr>
+              <th><a href="https://developer.apple.com/reference/appkit/nsaccessibility">AX</a></th>
+              <td>
+                <div class="general">Not mapped</div>
+              </td>
+            </tr>
+            <tr>
+              <th>Comments</th>
+              <td>
+                <p class="note">A button that represents a <a data-cite="html/forms.html#concept-submit-button">submit button</a> with a [=button/form owner=] cannot invoke a command.</p>
```

Comment:
This bit is missing the additional reset button mention that was added for popover section above.